### ISSUE TITLE
Add a new customise page 

### ISSUE THREAD

**Issue by rohan9024** [rohan9024/ecommerce#76]
![image](https://user-images.githubusercontent.com/64919642/208233376-228b638f-d4cb-4d37-a5f8-fb61ba5a3e5c.png)

The user should be able to customise his shoes, men shirts, women, kids [ add beta tag ]. 
Customise according to color for now 

You can refer this page -->

https://www.nike.com/in/u/custom-nike-dunk-unlocked-by-you-10001340/3603102466

**Comment by singhcoder694** [comment]
@rohan9024 The provided link is not opening....
<img width="960" alt="Screenshot 2023-12-08 110913" src="https://github.com/rohan9024/ecommerce/assets/114601400/fba3264b-048f-4eec-8549-a1dc8eb0aca8">

As well as I am facing issue while setting up locally. Can you help me with this...
It is showing...
**webpack compiled with 1 warning**
<img width="593" alt="Screenshot 2023-12-08 111018" src="https://github.com/rohan9024/ecommerce/assets/114601400/7a6bd85d-58a4-4a1f-81cd-d7d905c0fab6">
<img width="581" alt="Screenshot 2023-12-08 111033" src="https://github.com/rohan9024/ecommerce/assets/114601400/d13e8952-6811-44e4-905c-9afa271f868c">

It is running on localhost but showing a blank page...
<img width="960" alt="Screenshot 2023-12-08 110900" src="https://github.com/rohan9024/ecommerce/assets/114601400/e7944705-3a8a-4e92-bcba-2827909b14a3">

**Comment by singhcoder694** [comment]
Yes I have done this after that I started the server using **npm start** it started but with above errors

**Comment by singhcoder694** [comment]
Ok, please assign this to me... So that I can start working on this

**Comment by rohan9024** [comment]
@singhcoder694 Would you like to work on this ?

**Comment by singhcoder694** [comment]
I would but didn't understand what to do. As per my opinion:
1.) Add a Customize page .
2.) Add a color filter to this so that we can filter product on color basis.
But it can't be possible since product doesn't have this attribute so differentiating on color basis is not possible...
If I am understanding it wrong, elaborate the issue...

**Comment by rohan9024** [comment]
@singhcoder694 You can refer this page -->

https://www.nike.com/in/nike-by-you
